src/utils/api.js

```diff
@@ -62,7 +93,7 @@ export const getActionsByTimeOfDay = (actions) => {
     [NIGHT]: 0,
   };
   actions.forEach((action) => {
-    const actionHourOfDay = getActionHourOfDay(action);
+    const actionHourOfDay = getActionHourOfDay(action, dateKey);
```

Comment:
Here you seem to be passing `dateKey`, but from the function above, you pass `view`.

Reply:
Not sure what `dateKey` is holding here, then.

Reply:
Probably should be called `view`.

Reply:
ah yes very good catch. should be called `view` (although code was working correctly: in `ActionsByTimeOfDayChart.js`, `getActionsByTimeOfDay(actions, view)` is called, and that `view` is then passed to `getActionHourOfDay`)